dist/parse-tsconfig.d.ts

```diff
@@ -1,7 +1,7 @@
-import { RollupContext } from "./context";
-import { IOptions } from "./ioptions";
-export declare function parseTsConfig(context: RollupContext, pluginOptions: IOptions): {
-    parsedTsConfig: import("typescript").ParsedCommandLine;
-    fileName: string | undefined;
-};

```

Comment:
these look to be exactly the same, a little confused why it's showing as a diff? 🤔  
looks to be the same spacing too?

seems to have happened for several of the generated declarations

Reply:
self resolved as this PR was closed and this change doesn't exist in the other PRs